src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -262,6 +262,56 @@ FabricInfo * RetrieveCurrentFabric(CommandHandler * aCommandHandler)
     return Server::GetInstance().GetFabricTable().FindFabricWithIndex(index);
 }
 
+void FailSafeCleanup(FailSafeContext & failSafeContext)
+{
+    FabricInfo * fabricInfo = Server::GetInstance().GetFabricTable().FindFabricWithIndex(failSafeContext.GetFabricIndex());
```

Comment:
Suggest early exist if `FabricInfo` is nullptr, and it would signify a pretty bad problem...